simulation/revolve2/simulation/custom_vector/object/vector2.py

```diff
@@ -0,0 +1,206 @@
+"""A representation of a 2d Vector."""
```

Comment:
I prefer to drop the 'object' directory.
I think `from revolve2.simulation.vector2 import Vector2, unit, ...etc`
would be nice